templates/new_service/.github/workflows/build-and-deploy.yml

```diff
@@ -0,0 +1,94 @@
+name: Build and Deploy
+concurrency: build_and_deploy_${{ github.ref_name }}
+
+on:
+  push:
+   branches:
+    - main
+  pull_request:
+    branches:
+    - main
+    types: [opened, reopened, synchronize, labeled]
+
+permissions:
+  contents: write
+  deployments: write
+  packages: write
+  pull-requests: write
+
+env:
+  TERRAFORM_BASE: terraform/application
+  HEALTHCHECK_CMD: 'healthcheck/all'
+
+jobs:
+  build:
+    name: Build
+    env:
+      DOCKER_IMAGE: #DOCKER_REPOSITORY#
+    outputs:
+      docker-image-tag: ${{ steps.build-image.outputs.tag }}
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v4
+      name: Checkout
+
+    - name: Build and push docker image
+      id: build-image
+      uses: DFE-Digital/github-actions/build-docker-image@master
+      with:
+        github-token: ${{ secrets.GITHUB_TOKEN }}
+        context: .
+        docker-repository: ${{ env.DOCKER_IMAGE }}
+        max-cache: true
+        reuse-cache: true
+        snyk-token: ${{ secrets.SNYK_TOKEN }}
+
+  deploy-review-app:
+    name: Deployment To Review
+    concurrency: deploy_review_${{ github.event.pull_request.number }}
+    if: ${{ github.event_name == 'pull_request' && contains(github.event.pull_request.labels.*.name, 'deploy') }}
+    needs: [build]
+    environment:
+      name: review
+      url: ${{ steps.deploy_review.outputs.environment_url }}
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Deploy App to Review
+      id: deploy_review
+      uses: DFE-Digital/github-actions/deploy-to-aks@master
+      with:
+        azure-credentials:  ${{ secrets.AZURE_CREDENTIALS }}
+        github-token: ${{ secrets.GITHUB_TOKEN }}
+        environment: review
+        pr-number: ${{ github.event.pull_request.number }}
+        sha: ${{ needs.build.outputs.docker-image-tag }}
+        terraform-base: ${{ env.TERRAFORM_BASE }}
+        healthcheck: ${{ env.HEALTHCHECK_CMD }}
+        db-seed: true
+
+  deploy:
+    name: Deploy environments
+    if: ${{ success() && github.ref == 'refs/heads/main' }}
+    environment:
+      name: ${{ matrix.environment }}
+      url: ${{ steps.deploy_app.outputs.environment_url }}
+    needs: [build]
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        environment: [development,production]
+    steps:
+    - name: Deploy app to ${{ matrix.environment }}
+      id: deploy_app
+      uses: DFE-Digital/github-actions/deploy-to-aks@master
+      with:
```

Comment:
Can we include the google wip? At least a commented example. It doesn't need to be a template variable

Reply:
added